src/frontend/components/DashboardVideoLiveControlPane/inputs/DashboardVideoLiveWidgetTextAreaInput/index.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import { DashboardVideoLiveWidgetTextAreaInput } from '.';
+
+const setValueMock = jest.fn();
```

Comment:
As explain in the other PR, this should be a real function storing the value and then check if it's the one expected.

Reply:
done